subtrack.MAUI/Pages/CreateSubscription.razor

```diff
@@ -1,13 +1,36 @@
 ﻿@page "/CreateSubscription"
 @inject NavigationManager NavigationManager
 
-<h3>CreateSubscription</h3>
-<br/>
-<p>returnUrl: @ReturnUrl</p>
+<div style="display: flex; justify-content: space-between; padding-bottom: 20px;">
+    <button type="button" class="btn btn-outline-dark">Back</button>
+    <button type="button" class="btn btn-outline-dark">Save</button>
+</div>
+
+<EditForm Model="@subscription">
```

Comment:
Use the onvalidsubmit prop and add the event method to be called when a subs info is correctly inputted. Call ISubService.Create and after that navigate to the newly created sub's detail page